ontimize/components/form/o-form.component.ts

```diff
@@ -181,6 +207,8 @@ export class OFormComponent implements OnInit, OnDestroy, CanComponentDeactivate
   stayInRecordAfterInsert: boolean = false;
   @InputConverter()
   stayInRecordAfterEdit: boolean = false;
+  @InputConverter()
```

Comment:
You should not use InputConverter here, because its only valid for boolean, string or number inputs

Reply:
Ok, this error was related to one change I made because of Manuel comments. I forgot to remove it, now it´s fine.